outlook/src/taskpane/components/Crm/LeadList/LeadListItem.tsx

```diff
@@ -18,13 +19,21 @@ const LeadListItem = (props: LeadsListItemProps) => {
         window.open(url,"_blank");
     }
 
-    let expectedRevenueString = props.lead.expectedRevenue+" at "+props.lead.probability+"%";
+    let expectedRevenueString = _t("%(expected_revenue)s at %(probability)s%", {
+        expected_revenue: props.lead.expectedRevenue,
+        probability: props.lead.probability
+    });
 
     if (props.lead.recurringPlan)
     {
-        expectedRevenueString = props.lead.expectedRevenue+ " + "+props.lead.recurringRevenue
-            +" "+props.lead.recurringPlan
-            +" at "+props.lead.probability+"%";
+        expectedRevenueString = _t("%(expected_revenue)s + %(recurring_revenue)s %(recurring_plan)s at %(probability)s%",
+                {
```

Comment:
Linting: Isn't it some indentation issue ?

Reply:
Yes, corrected!